doc/index.rst

```diff
@@ -99,8 +99,8 @@ you to cite the Pastas article published in Groundwater journal as follows:
 
     User Guide <userguide/index>
     Examples <examples/index>
-    API Docs <api/index>
+    Reference <api/index>
```

Comment:
Maybe we can call this API Reference just like like Pandas/Numpy/Scipy?

Reply:
Good idea will do, same for the other item (contribute > development)